chains/ethereum/config/src/lib.rs

```diff
@@ -258,6 +273,11 @@ pub fn config(network: &str) -> anyhow::Result<BlockchainConfig> {
         "arbitrum" => return arbitrum_config("mainnet"),
         "arbitrum-goerli" => return arbitrum_config("goerli"),
 
+        // Binance
+        "binance-dev" => return binance_config("dev"),
+        "binance" => return binance_config("mainnet"),
+        "Testnet" => return binance_config("testnet"),
```

Comment:
Let's keep the lowercase as standard.
```suggestion
        "testnet" => return binance_config("testnet"),
```